detectron2/layers/csrc/box_iou_rotated/box_iou_rotated_utils.h

```diff
@@ -93,7 +93,7 @@ HOST_DEVICE_INLINE int get_intersection_points(
   // while it can cause drastic difference if we miss an intersection/vertex.
   // Therefore, we add an epsilon to relax the comparisons between
   // the float point numbers that decide the intersection points.
-  double EPS = 1e-5;
```

Comment:
Shouldn't it be changed to `T` instead?

Reply:
You are correct. Updated and manually tested that the speed is the same.